recon/src/protocol.rs

```diff
@@ -931,33 +925,12 @@ impl<S> SinkFlusher<S> {
         self.feed_count = 0;
         Ok(())
     }
-    async fn feed<T, E>(&mut self, message: T) -> Result<()>
```

Comment:
Is it possible that the additional sending is causing the additional sawing in the graph? I assumed that things are syncing faster, so we have to understand more ranges, but I'm wondering if there's somewhere that sending more/less together might help get bigger overlaps. 

It's also possible that we just didn't have tests demonstrating this time/number of keys synced, so we just eventually got to really fast 0 cases.

Reply:
I deleted this to simplify the api and avoid having to think about feed/flush.. but now you just have to collect in memory to send_all so it doesn't seem like an improvement.